setup.py

```diff
@@ -157,6 +157,7 @@ def get_git_sha() -> str:
         "snowflake": [
             "snowflake-sqlalchemy==1.2.4"
         ],  # PINNED! 1.2.5 introduced breaking changes requiring sqlalchemy>=1.4.0
+        "sybase": ["sqlalchemy-sybase>=1.0.6"],
```

Comment:
nit: Let's restrict this to the current major version:
```suggestion
        "sybase": ["sqlalchemy-sybase>=1.0.6, <2"],
```